service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/entity/Trip.kt

```diff
@@ -0,0 +1,12 @@
+package org.thechance.service_taxi.domain.entity
+
+data class Trip(
+    val id: String,
+    val driverId: String? = null,
+    val clientId: String? = null,
+    val from: Pair<Long, Long>? = null,
+    val to: Pair<Long, Long>? = null,
+    val rate: Double? = null,
+    val price: Double? = null,
+    val isDeleted: Boolean? = null,
```

Comment:
i think `isDeleted `could just have default value as false and not be nullable

Reply:
I got error in update so i did it like that